extensions/spotify-player/src/api/play.ts

```diff
@@ -25,7 +25,7 @@ export async function play({ id, type, contextUri }: PlayProps = {}) {
 
   try {
     if (!type || !id) {
-      await spotifyClient.putMePlayerPlay();
+      await spotifyClient.putMePlayerPlay({ context_uri: contextUri });
```

Comment:
is this safe? what if `contextUri` is undefined?

Reply:
I just tested the uses of `play` that don't provide `contextUri`, and everything seems to work fine. I think calling `putMePlayerPlay` when `contextUri` is undefined is no different than not providing it at all.